<launch>
  <group ns="plug_detector" clear_params="true">
    <param name="display" type="bool" value="true"/>
    <param name="roi_policy" type="str" value="LastImageLocation"/>
    <!-- Plug checkerboard -->
    <param name="square_size" type="double" value="0.0042"/>
    <param name="board_width" type="int" value="3"/>
    <param name="board_height" type="int" value="4"/>
    <!-- Calibration checkerboard -->
    <!--param name="square_size" type="double" value="0.0215"/>
    <param name="board_width" type="int" value="6"/>
    <param name="board_height" type="int" value="9"/-->
  </group>
  <node pkg="outlet_detection" type="plug_node" respawn="false" output="screen"/>
</launch>
